src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue

```diff
@@ -256,7 +259,7 @@ export default {
 					summary.push(this.getDisplayNameForReaction(list[item]))
 				}
 			}
-			return summary.join(', ')
+			return summary.join(', ') + (this.reactionsCount(reaction) > 3 ? ', ' : '')
```

Comment:
Comma before `and` in sentences like `First, second and third` is not used in:
- Russian
- British English
- German
- French (from internet)
- Arabic (from internet)
Exception: American English

I'd say, we drop this change? Or provide a complete translatable string for 1,2,3,4+ options?

Reply:
Let's drop it